extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -72,6 +72,18 @@ namespace Microsoft.Windows.DevHome.SDK
         Windows.Foundation.IAsyncOperation<ProviderOperationResult> CloneRepositoryAsync(IRepository repository, String cloneDestination, IDeveloperId developerId);
     };
 
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 2)] interface IRepositoryProvider2
+    requires IRepositoryProvider
+    {
+        String[] SearchFieldNames
+        {
+            get;
+        };
+
+        Windows.Foundation.IAsyncOperation<IVectorView<String> > GetValuesForSearchFieldAsync(String requestedSearchField, IMapView<String, String> fieldValues, IDeveloperId developerId);
```

Comment:
Super nit:
```suggestion
        Windows.Foundation.IAsyncOperation<IVectorView<String>> GetValuesForSearchFieldAsync(String requestedSearchField, IMapView<String, String> fieldValues, IDeveloperId developerId);
```

Reply:
Going to leave this be.  The double `>>` causes a (syntax error in IDLs.)[https://learn.microsoft.com/en-us/uwp/midl-3/troubleshooting]